src/main/java/duke/frontend/MainWindow.java

```diff
@@ -0,0 +1,79 @@
+package duke.frontend;
+
+import duke.Duke;
+import duke.Ui;
+import duke.exceptions.InvalidTaskException;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private boolean isExit = false;
```

Comment:
Variable names are in camelCase, follows the naming conventions well. Boolean variable is also named to sound like a boolean. Great!